tests/unit/dataactvalidator/test_b15_object_class_program_activity.py

```diff
@@ -11,8 +11,10 @@ def test_success(database):
     """ Tests that SF 133 amount sum for line 2104 matches the calculation from Appropriation based on the fields below
         for the specified fiscal year and period """
     tas = "".join([_TAS, "_success"])
+    tas2 = "".join([_TAS, "_other_tas"])
```

Comment:
I don't really have strong preferences here, but I know in the past we've changed away from using '+' for string concatenation so I've generally been leaning towards either format or join.